css/media.css

```diff
@@ -109,38 +134,60 @@
         margin-bottom: 0;
     }
 
+
+     /* animation section hidden */
+    .container-a {
+        display: none;
```

Comment:
I noticed there are a few properties you have defined in both the tablet and mobile CSS. I wonder if you could create a media section that covers both sizes and put the properties common to both in those. Just a thought. I'm not sure if it works that way.